editor/scene_tree_dock.cpp

```diff
@@ -1620,6 +1621,56 @@ void SceneTreeDock::_selection_changed() {
 	_update_script_button();
 }
 
+void SceneTreeDock::_do_create(Node* p_parent) {
```

Comment:
"Pointer and reference operators are affixed to the variable identifier, not to the type name."

https://travis-ci.org/godotengine/godot/jobs/425751969#L533
http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html

Reply:
Fixed